packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1006,6 +1007,14 @@ export class GarbageCollector implements IGarbageCollector {
 			return;
 		}
 
+		if (!toNodePath.startsWith("/")) {
```

Comment:
Shouldn't we validate that fromNodePath is also absolute? For that one, we can assert I believe.

Reply:
It hits an assert later, actually - that's how I knew to put the `""` prefix in the array when building it up.

But in the interest of failing fast I'll add one here.  It was kind of obscure (something asserted it wasn't NodeType.Other I think)